resources/sdk/purecloudjava-guest/templates/README.mustache

```diff
@@ -150,6 +150,106 @@ Specify the connector in the builder:
 .withProperty(ApiClientConnectorProperty.CONNECTOR_PROVIDER, new OkHttpClientConnectorProvider())
 ```
 
+#### SDK Logging
+
+Logging of API requests and responses can be controlled by a number of parameters on the `Configuration`'s `Logger` instance.
+
+`LogLevel` values:
+1. trace (HTTP Method, URL, Request Body, HTTP Status Code, Request Headers, Response Headers)
+2. debug (HTTP Method, URL, Request Body, HTTP Status Code, Request Headers)
+3. error (HTTP Method, URL, Request Body, Response Body, HTTP Status Code, Request Headers, Response Headers)
+4. none - default
+
+`LogFormat` values:
+1. JSON
+2. Text - default
+
+By default, the request and response bodies are not logged because these can contain PII. Be mindful of this data if choosing to log it.  
+To log to a file, provide a value to `setLogFilePath`. SDK users are responsible for the rotation of the log file.
+
+Example logging configuration:
+```{"language":"java"}
+ApiClient.LoggingConfiguration loggingConfiguration = new ApiClient.LoggingConfiguration();
+loggingConfiguration.setLogLevel("trace");
+loggingConfiguration.setLogFormat("json");
+loggingConfiguration.setLogRequestBody(true);
+loggingConfiguration.setLogResponseBody(true);
+loggingConfiguration.setLogToConsole(true);
+loggingConfiguration.setLogFilePath("/var/log/javasdk.log");
+
+ApiClient apiClient = ApiClient.Builder.standard().withLoggingConfiguration(loggingConfiguration).build();
+```
+
+#### Configuration file
+
+A number of configuration parameters can be applied using a configuration file. There are two sources for this file:
+
+1. The SDK will look for `%HOMEDRIVE%%HOMEPATH%\.genesyscloudjava\config` on Windows, or `$HOME/.genesyscloudjava/config` on Unix.
+2. Provide a valid file path to the ApiClient
+
+Example setting the configuration file:
+
+```{"language": "java"}
+ApiClient apiClient = ApiClient.Builder.standard().withConfigFilePath("/path/config").build();
+```
+
+The SDK will check every 30 seconds to see if the config file has been updated, regardless of whether a config file was present at start-up. To disable this behaviour, set `autoReloadConfig` to false.  
+Example setting autoReloadConfig to false:
+
+```{"language": "java"}
+ApiClient apiClient = ApiClient.Builder.standard().withAutoReloadConfig(false).build();
+```
+INI and JSON formats are supported. See below for examples of configuration values in both formats:
+
+INI:
+```{"language":"ini"}
+[logging]
+log_level = trace
+log_format = text
+log_to_console = false
+log_file_path = /path/log
+log_response_body = false
+log_request_body = false
+[retry]
```

Comment:
You can remove retry and reauthentication from this and the JSON example since it doesn't apply to this SDK